operator/builtin/transformer/ratelimit/rate_limit.go

```diff
@@ -84,7 +84,7 @@ func (p *RateLimitOperator) Start() error {
 
 	ctx, cancel := context.WithCancel(context.Background())
 	p.cancel = cancel
-	p.wg = &sync.WaitGroup{}
+	p.wg = sync.WaitGroup{}
```

Comment:
Now that this is not a pointer, no need to initialize it -- the zero value of `sync.WaitGroup` is valid